apps/server/src/shared/infra/collaborative-storage/collaborative-storage.adapter.spec.ts

```diff
@@ -59,4 +61,11 @@ describe('CollaborativeStorage Adapter', () => {
 			);
 		});
 	});
+
+	describe('Delete Team from Nextcloud', () => {
+		it('should call the strategy', () => {
+			const teamIdMock = 'teamIdMock';
+			adapter.deleteTeam(teamIdMock);
```

Comment:
there seems to be an expect missing here... see above

Reply:
now they are no longer missing